File: 0215-kth-largest-element-in-an-array/0215-kth-largest-element-in-an-array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> pq = new PriorityQueue<>(Collections.reverseOrder()); // comparator for maxHeap
        for(int i : nums) pq.offer(i);
        for(int i=1; i<k; i++){
            pq.poll();
        }
        return pq.poll();
    }
}

File: 0215-kth-largest-element-in-an-array/0215-kth-largest-element-in-an-array.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        priority_queue<int, vector<int>, greater<int>> pq;
        for(int i=0; i<k; i++) pq.push(-10001);
        for(int num : nums){
            if(num > pq.top()){
                pq.pop();
                pq.push(num);
            }
        }
        return pq.top();
    }
};